src/configuration/configuration.ts

```diff
@@ -0,0 +1,40 @@
+import { ethers } from "ethers";
+import * as zAuction from "@zero-tech/zauction-sdk";
+
+export class Configuration implements zAuction.Config{
```

Comment:
This class name is confusing since I thought it was a `zns` sdk configuration class and not a zAuction configuration class.

If this is only used for zAuction, make sure the name implies it. Additionally, what functionality does this class provide which a generic object casted with `zAuction.Config` interface does not? It appears that it only helps by defaulting some fields as `""` (default values) however those default values aren't helpful since if you try to create a `zAuction.Instance` with empty config values it will most likely fail (depending on which config value)

Assume that all 5 parameters are needed. And since they are, having a class like this doesn't provide much helper functionality.

---
Talked about this over phone call